consts.ts

```diff
@@ -3,75 +3,268 @@ export type Lang = "en" | "sk";
+        "Dobývanie rúd",
+        "Štôlňa Lászlo Vashegyi",
+        "Výrobky domáceho drevárskeho priemyslu a ťažba dreva",
+        "Drevené náradie a nástroje",
+      ],
     },
   },
   en: {
     sng: "Slovak National Gallery",
-    igorGazdikCollection: "Selection from the book collection of Igor Gazdík",
-    annotation: [
-      "Pavel Dobšinský: Prostonárodné slovenské povesti (Slovak Tales), notebook 1. Turčiansky sv. Martin : Kníhtlačiarsky účastinársky spolok, 1922. Third edition. Illustrations Andrej Mihal. SNG, UP-P 3280/30a",
-      "Pavel Dobšinský: Prostonárodné slovenské povesti (Slovak Tales), notebook 9. Turčiansky sv. Martin : Kníhtlačiarsky účastinársky spolok, 1922. Second edition. Illustrations  Mikuláš Galanda. SNG, UP-P 3280/30b",
-      "Ivan Horváth: Strieborný prach (Silver Dust). Bratislava : Slovenská grafia, 1929. Typography Karel Jaroň. SNG, UP-P 3280/41",
-      "Jaroslav Seifert: Na vlnách TSF (On the Waves of TSF). Prague : Václav Petr, 1925. Hosta Edition, Volume 1. Cover and typography Karel Teige. SNG, UP-P 3280/3649",
-      "Štefan Janšák: Lovci hlinených perál (The Hunters of Clay Pearls). Bibliophile edition. 1934. Print: Linografia K. Jaroň a spol., Bratislava. Illustrations: Mikuláš Galanda. Graphic design: Karol Jaroň, UP-P 3280/809",
-    ],
-    description: [
-      "The exposition Biblioman brings you the joy that stems from the reading experience and visual perception of bibliophile books also through digitized works from Igor Gazdík's collection.",
-      "The first volume of the selected digitized titles focuses on Slovak and Czech books. It showcases typographic and illustrative creations from the beginning of book design and bibliophilia in Slovakia, featuring its several protagonists, such as Ľudovít Fulla, Mikuláš Galanda, and Karol Jaroň.",
-      "In this series of digitized books, visitors can dive into Slovak classics like the Fairy Tales by Pavel Dobšinský, Ivan Horváth's novella or essays by Štefan Janšák from the beginnings of Slovak archaeology. Czech bibliophilia is represented by the legendary collection of poems by Jaroslav Seifert in the equally masterful graphic/typographic design by Karol Teige.",
-    ],
     library: {
       "1": {
-        title: "Silver Dust",
-        author: "Ivan Horváth",
-        year: "1929",
+        title: "Bardejovské Kúpele in Pictures",
+        author: "Kornél Chyzer – Károly Divald",
+        year: "1886",
+        photosTitles: [
+          "Bardejovské Kúpele in Pictures",
+          '"View of Bardejovské Kúpele from the southeast"',
+          "Bardejovská Street",
+          "Church Square",
+          "Main spring with spa hall",
+          "Main Street",
+          "Main Street",
+          "Pond and City Hotel",
+          '"Church Square with new spa house"',
+          '"Hot spring with old spa house"',
+          '"Sweet Spring"',
+          "Sulphur spring",
+          '"Faith, Love, and Hope" memorial',
```

Comment:
```suggestion
          '"Faith, Love, and Hope" memorial"',
```

? missing last apostrophe